cli/azd/pkg/project/service_manager.go

```diff
@@ -583,8 +583,14 @@ func (sm *serviceManager) GetFrameworkService(ctx context.Context, serviceConfig
 	// project that handles the containerization.
 	requiresLanguage := serviceConfig.Language != ServiceLanguageDocker && serviceConfig.Language != ServiceLanguageNone
 	if serviceConfig.Host.RequiresContainer() && requiresLanguage {
+		var serviceTargetType = string(ServiceLanguageDocker)
+
+		if serviceConfig.Language == ServiceLanguageDotNet {
```

Comment:
I wonder if we want to provide a way for folks to get the old behavior (where we'd wrap this in a DockerProject which would use a `Dockerfile`).  Maybe if there was a `Dockerfile` at the root of the project folder?

Reply:
I really love the decision here to pick `dotnet publish` over Oryx build. I could see `docker build` if `Dockerfile` was set explicitly, or `Dockerfile` is unset but present in the project.